packages/elements-react/src/components/form/form.tsx

```diff
@@ -267,11 +267,10 @@ export function OryForm({
     }
 
     return (
-      <div className="grid gap-8" data-testid={dataTestId}>
+      <div data-testid={dataTestId}>
```

Comment:
TODO: this file is not part of the defaultTheme, but renders a DOM node. That should not happen! 

It was here before, so not related to your change, but something to keep in mind (I'll try to improve the tooling for this once everything is done).